src/darwin/CHIPTool/CHIPTool/View Controllers/Fabric/FabricUIViewController.m

```diff
@@ -223,6 +241,20 @@ - (void)fetchFabricsList
 
 // MARK: UIButton methods
 
+- (IBAction)removeAllFabricsButtonPressed:(id)sender
```

Comment:
Suggest adding logic for a confirmation dialog, in case this stays for longer than just development, as it is a factory reset. Factory resets are very disruptive and usually come with a coinfirmation.

Reply:
I'll add a confirmation/cancel dialog explanation what RemoveAllFabrics does